src/test/java/tdl/record/sourcecode/content/CopyFromDirectorySourceCodeProviderTest.java

```diff
@@ -0,0 +1,38 @@
+package tdl.record.sourcecode.content;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import static org.junit.Assert.assertFalse;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import tdl.record.sourcecode.test.FileTestHelper;
+
+public class CopyFromDirectorySourceCodeProviderTest {
```

Comment:
While you are here, it might be worth adding the test for the other behaviours too. ( not copying .git )

Reply:
Got it! I put the assertion for it in the same test method.

Reply:
https://github.com/julianghionoiu/dev-sourcecode-record/pull/6/files#diff-5cd7a7c4506348698934ef5c5c51e9feR44